modmesh/pilot/airfoil/__init__.py

```diff
@@ -30,7 +30,11 @@
 """
 
 from ._naca import Naca4, Naca4Sampler
-from ._airfoil_gui import Naca4Airfoil
+from .. import _pilot_core as _pcore
+if _pcore.enable:
```

Comment:
Avoid importing GUI code directly in the package `modmesh.pilot.airfoil` namespace.